package.json

```diff
@@ -64,21 +64,17 @@
     "npm-merge-driver-install": "^1.0.0",
     "npm-run-all": "^4.1.3",
     "pkg-ok": "^2.2.0",
-    "rollup": "^0.65.0",
+    "rollup": "^2.2.0",
     "shx": "^0.3.2",
     "sinon": "^6.1.5",
-    "videojs-generate-karma-config": "^5.1.0",
-    "videojs-generate-rollup-config": "^3.1.0",
+    "videojs-generate-karma-config": "^8.0.1",
```

Comment:
Also update version of this package which has [removed](https://github.com/videojs/videojs-generate-karma-config/pull/40) older browserstack launchers